app/src/main/java/org/greenstand/android/TreeTracker/view/SelectableItemDetail.kt

```diff
@@ -84,48 +84,8 @@ fun UserButton(
     selectedColor: Color,
     onClick: () -> Unit
 ) {
-    SelectableImageDetail(
-        user.photoPath,
-        isSelected,
-        buttonColors,
-        selectedColor,
-        onClick
-    ) {
-        Text(
-            text = "${user.firstName} ${user.lastName}\n${user.wallet}",
-            color = AppColors.LightGray,
-            fontSize = 12.sp,
-            fontWeight = FontWeight.SemiBold,
-            fontFamily = FontFamily.SansSerif,
-        )
-        Row(
-            modifier = Modifier.padding(top = 4.dp),
-            verticalAlignment = Alignment.CenterVertically
-        ) {
-            Box( // TODO: Change into 'Plant' icon.
-                modifier = Modifier
-                    .size(width = 20.dp, height = 22.dp)
-                    .background(AppColors.LightGray)
-            )
-            Text(
-                text = "1,234", // TODO: Fetch user's token count.
-                modifier = Modifier.padding(start = 4.dp),
-                color = AppColors.LightGray,
-                fontSize = 16.sp,
-                fontWeight = FontWeight.SemiBold,
-                fontFamily = FontFamily.SansSerif,
-            )
-        }
-    }
-}
-@Composable
-fun WalletUserButton(
-    user: User,
-    isSelected: Boolean,
-    buttonColors: DepthButtonColors,
-    selectedColor: Color,
-    onClick: () -> Unit
-) {
+    val credential: String? = if (user.email.isNullOrEmpty()) user.phone else user.email
```

Comment:
This should be the users "wallet". Their wallet is the ID that the account is associated with. Atm this is an email/phone but in the future it could be something else such as "@myWalletName".

Reply:
This is subject to your response in https://github.com/Greenstand/treetracker-android/pull/674#discussion_r748724697